server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/CompactionStrategy.java

```diff
@@ -28,11 +28,11 @@
 /**
  * The interface for customizing major compactions.
  * <p>
- * Important: Compaction configurations should use one compaction scheme, either the
- * {@link CompactionPlanner} introduced in 2.1, or a CompactionStrategy (see deprecation section).
+ * Important: Compaction configurations should use one compaction scheme, either the new
```

Comment:
Even when using the CompactionStrategy, parts of the new scheme (like the planner) will still come into play because we can not have two separate implementations. Not sure of the best way to word it but need to convey the following.

 * Continued use of the CompactionStrategy is suboptimal
 * Continued support for the CompactionStrategy was only provided to give time to migrate to the new and improved plugins.  Users need to migrate ASAP in order to have a better experience.
 * The CompactionStrategy will be removed in the next major release.